### PR TITLE
Feat/add dream properties

### PR DESCRIPTION
**Pull request by basels**

### CODE REVIEW

File: app/views/camps/_form.haml [hunk 1]
```diff
@@ -114,6 +129,95 @@
...
+      = form.label t("form_dream_prop_community_is_installation_present_for_public")
+      = form.check_box :dreamprop_community_is_installation_present_for_public
+
+    .combo
+      = form.label t("form_dream_prop_community_is_context")
+      = form.check_box :dreamprop_community_is_context
+
+    .combo
+      = form.label t("form_dream_prop_community_conext_desc")
+      = form.text_field :dreamprop_community_conext_desc, :max_length => 4096 #3.4.3.1 TODO: only if previous flag
```
Comment (rootux) [inline app/views/camps/_form.haml]: @basels not sure what is this todo? is it still relevant?

File: app/views/camps/_form.haml [hunk 2]
```diff
@@ -244,13 +348,13 @@
       = form.text_area :budgetplan, :maxlength => 4096
       = t("form_budgetplan_guidetext_html")
 
-  - if ( (Rails.configuration.x.firestarter_settings["granting"] and !Rails.application.config.x.firestarter_settings['disable_edit_budget']) or (current_user && current_user.guide || current_user.admin))
+  - if Rails.configuration.x.firestarter_settings["granting"] and !Rails.application.config.x.firestarter_settings['disable_edit_budget']
```
Comment (rootux) [inline app/views/camps/_form.haml]: you've removed some code that I would keep regarding user and admin

File: app/views/camps/_form.haml [hunk 3]
```diff
@@ -244,13 +348,13 @@
...
     .combo#minbudget_realcurrency
       = form.label t("form_minigrants_realcurrency_label")
       = form.number_field :minbudget_realcurrency, :maxlength => 6 ,class:'form-control'
       = t("form_minigrants_realcurrency_guidetext_html")
 
-  - if ( (Rails.configuration.x.firestarter_settings["granting"] and !Rails.application.config.x.firestarter_settings['disable_edit_budget']) or (current_user && current_user.guide || current_user.admin))
```
Comment (rootux) [inline app/views/camps/_form.haml:270]: same here - I would undo this one

**Review by rootux — COMMENTED** [review]
Please review

File: app/views/camps/_form.haml [hunk 4]
```diff
@@ -11,6 +11,21 @@
...
+    if($("#camp_dreamprop_community_is_context").is(':checked')) {
+        $("#community_context").show();
+    } else {
+        $("#community_context").hide();
+    }
+    if($("#camp_dreamprop_theme_annual").is(':checked')) {
+        $("#theme_annual").show();
+    } else {
+        $("#theme_annual").hide();
+    }
```
Comment (rootux) [inline app/views/camps/_form.haml:28]: this place is DRY... :)
would replace it soon with some generic js code

Comment (basels) [reply app/views/camps/_form.haml:28]: will do a small research and fix during next tasks :)

File: README.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
-# Midburn Dreams
+# Midburnerot Dreams
```
Comment (rootux) [inline README.md:1]: @basels nope - I believe those also don't belong here

**Pull request merged**